ui/raidboss/data/03-hw/dungeon/sohm_al.js

```diff
@@ -6,6 +6,27 @@ import ZoneId from '../../../../../resources/zone_id.js';
 // Sohm Al (normal)
 export default {
   zoneId: ZoneId.SohmAl,
+  timelineFile: 'sohm_al.txt',
+  timelineTriggers: [
+    {
+      id: 'Sohm Al Raskovnik Bloody Caress',
```

Comment:
Just checking, but none of these three have cast bars?

Reply:
Yes, they are instant skills.
Although there is some special animations before them, time is not enough to use damage reducing abilities.
Besides, they are not very hurt for a high-item-lv player, but if your ilv is low, these deliver about half hp, so I made them.